Misc/NEWS.d/next/Library/2018-09-27-19-31-47.bpo-34822.EztBhL.rst

```diff
@@ -0,0 +1,5 @@
+Simplified AST for subscription. Simple indices will be represented by their
+value, extended slices will be represented as tuples.  :mod:`ast` classes
+``Index`` and ``ExtSlice`` are considered deprecated and will be removed in
+future Python versions. In meanwhile, ``Index(value)`` will return a ``value``
+itself, ``ExtSlice(slices)`` will return ``Tuple(slices, Load())``.
```

Comment:
Maybe replace "will be" with "are now"? Also change "will return" into "now returns". (But "will be removed" must stay, since it is truly in the future.)